### DISCUSSION TITLE
Realtime file changes detection?

### DISCUSSION THREAD

**Discussion by ghost** [WinMerge/winmerge#756] [Q&A] [answered]
Is there a way to use only one file. To detect the changes after and before without to copy the file two times?

**Comment by sdottaka** [comment, marked as answer]
In version 2.16.12, when comparing a single file in the 'Select Files or Folders window', WinMerge will now automatically copy the file to a temporary folder and compare it with the original file.

Although not in real-time, when WinMerge regains focus after another application has updated the original file, a message box will appear asking if the file should be reloaded.

To specify this from the command line, specify /self-compare
